test/testdata/generate.sh

```diff
@@ -58,58 +48,55 @@ scaffold_test_project() {
     $kb create api --group crew --version v1 --kind Laker --controller=true --resource=false --make=false
-    if [ $project == "project-v3" ]; then
-      $kb create webhook --group crew --version v1 --kind Captain --defaulting --programmatic-validation --force
-    fi
   elif [[ $project =~ multigroup ]]; then
     header_text 'Switching to multigroup layout ...'
     $kb edit --multigroup=true
 
     header_text 'Creating APIs ...'
     $kb create api --group crew --version v1 --kind Captain --controller=true --resource=true --make=false
-    $kb create webhook --group crew --version v1 --kind Captain --defaulting --programmatic-validation
+    $kb create webhook --group crew --version v1 --kind Captain --defaulting --programmatic-validation --make=false
     $kb create api --group ship --version v1beta1 --kind Frigate --controller=true --resource=true --make=false
-    $kb create webhook --group ship --version v1beta1 --kind Frigate --conversion
+    $kb create webhook --group ship --version v1beta1 --kind Frigate --conversion --make=false
     $kb create api --group ship --version v1 --kind Destroyer --controller=true --resource=true --namespaced=false --make=false
-    $kb create webhook --group ship --version v1 --kind Destroyer --defaulting
+    $kb create webhook --group ship --version v1 --kind Destroyer --defaulting --make=false
     $kb create api --group ship --version v2alpha1 --kind Cruiser --controller=true --resource=true --namespaced=false --make=false
-    $kb create webhook --group ship --version v2alpha1 --kind Cruiser --programmatic-validation
+    $kb create webhook --group ship --version v2alpha1 --kind Cruiser --programmatic-validation --make=false
     $kb create api --group sea-creatures --version v1beta1 --kind Kraken --controller=true --resource=true --make=false
     $kb create api --group sea-creatures --version v1beta2 --kind Leviathan --controller=true --resource=true --make=false
     $kb create api --group foo.policy --version v1 --kind HealthCheckPolicy --controller=true --resource=true --make=false
     $kb create api --group apps --version v1 --kind Pod --controller=true --resource=false --make=false
     if [ $project == "project-v3-multigroup" ]; then
       $kb create api --version v1 --kind Lakers --controller=true --resource=true --make=false
-      $kb create webhook --version v1 --kind Lakers --defaulting --programmatic-validation
+      $kb create webhook --version v1 --kind Lakers --defaulting --programmatic-validation --make=false
     fi
   elif [[ $project =~ addon ]]; then
     header_text 'enabling --pattern flag ...'
     export KUBEBUILDER_ENABLE_PLUGINS=1
     header_text 'Creating APIs ...'
-    $kb create api --group crew --version v1 --kind Captain --controller=true --resource=true --pattern=addon
+    $kb create api --group crew --version v1 --kind Captain --controller=true --resource=true --make=false --pattern=addon
```

Comment:
btw, why you add the make=false in the above instructions? Was it done to solve so problem faced after the changes?

Reply:
There is no reason to compile the manager in the testdata folder and then remove it. There is no reason to go fmt and go vet all the directory for every creat call, etc. We already chose to set `--make=false` but some resources that were added after or webhooks which didnt have that flag previously still didnt have it, I just added it to the remaining ones.